### DISCUSSION TITLE
github.dev doesn't sign commits

### DISCUSSION THREAD

**Discussion by davorpa** [community/community#8174] [Codespaces] [answered]
Enabling new beta feature about show a badge about unverified commits I realize that those made from web editor doesn't sign commits like inline single file editor does.

Then the verified badge is not shown.

Also happens when PR suggestions are used to solve conflicts or make a fast review changes

**Comment by Katsute** [comment, marked as answer]
https://github.com/github/feedback/discussions/5841#discussioncomment-1614744

**Reply by davorpa** [reply]
Thanks @Katsute to point me to the dup. I can't find same question at a glance
